Correct "Durable Task extension"

andynava opened the pull request:
"Durable Task extension" -> "Durable Functions extension"

Extension is called Durable Functions extension, not Durable Task extension.

From documentation (https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-overview#the-technology):
"Durable Functions extension is built on top of the Durable Task Framework"

azurecla commented:
Hi __@abaranch__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

deneha commented:
@abaranch Thank you for this contribution.  I'll have the author review.

alancameronwills commented:
Thanks, Anastasia.
Dene, please pull.

andynava committed:
Correct "Durable Task extension"

"Durable Task extension" -> "Durable Functions extension"

Extension is called Durable Functions extension, not Durable Task extension.

From documentation (https://docs.microsoft.com/en-us/azure/azure-functions/durable-functions-overview#the-technology):
"Durable Functions extension is built on top of the Durable Task Framework"
